website/docs/docs/cloud/migration.md

```diff
@@ -7,34 +7,45 @@ pagination_next: null
 pagination_prev: null
 ---
 
-dbt Labs is in the process of migrating dbt Cloud to a new _cell-based architecture_. This architecture will be the foundation of dbt Cloud for years to come, and will bring improved scalability, reliability, and security to all customers and users of dbt Cloud.
+dbt Labs is in the process of rolling out a new cell-based architecture for dbt Cloud. This architecture will be the foundation of dbt Cloud for years to come, and brings improved reliability, performance, and consistency to users of dbt Cloud.
 
-There is some preparation required to ensure a successful migration.
+Migrations are being scheduled on a per-account basis. If you have yet to receive any communication (either with a banner or by email) about a migration date, you don't need to take any action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
 
-Migrations are being scheduled on a per-account basis. _If you haven't received any communication (either with a banner or by email) about a migration date, you don't need to take any action at this time._ dbt Labs will share migration date information with you, with appropriate advance notice, before we complete any migration steps in the dbt Cloud backend.
+The migration process has been automated to the extent possible. However, you must take action to complete it. This document outlines the steps you must take to prevent service disruptions when your account is migrated to the cell-based architecture. 
```

Comment:
```suggestion
Your account will be automatically migrated on its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
```